Makefile

```diff
@@ -110,6 +112,10 @@ webextension/build/inlineSelectionCss.js: build/server/static/css/inline-selecti
 	@mkdir -p $(@D)
 	./bin/build-scripts/css_to_js inlineSelectionCss $< > $@
 
+webextension/build/raven.js: $(raven_source)
+	@mkdir -p $(@D)
+	cp $< $@
```

Comment:
It looks like we're getting some ESLint errors:

```
/home/travis/build/mozilla-services/pageshot/webextension/build/raven.js
    13:128  error    'define' is not defined                                              no-undef
    13:140  error    'define' is not defined                                              no-undef
    13:337  warning  'define' is defined but never used                                   no-unused-vars
    13:344  warning  'module' is defined but never used                                   no-unused-vars
    13:351  warning  'exports' is defined but never used                                  no-unused-vars
  1953:2    error    Unnecessary semicolon                                                no-extra-semi
  2172:21   error    'groups' is already defined                                          no-redeclare
  2338:14   warning  'escapeCodeAsRegExpForMatchingInsideHTML' is defined but never used  no-unused-vars
  2395:13   warning  'reference' is assigned a value but never used                       no-unused-vars
  2407:25   error    Expected a conditional expression and instead saw an assignment      no-cond-assign
  2516:13   warning  'source' is defined but never used                                   no-unused-vars
```

Not sure if this path needs to be added to .eslintignore [since it seems to be copied from _/node_modules/_].